spring-cloud-task-samples/taskprocessor/README.adoc

```diff
@@ -0,0 +1,44 @@
+= TaskProcessor
```

Comment:
We should mention in the doc the dependencies that this module has (redis in this case).